package.json

```diff
@@ -54,15 +54,15 @@
     "mozilla",
     "shield-study"
   ],
-  "license": "MIT",
+  "license": "MPL-2.0",
   "main": "bootstrap.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/biancadanforth/tracking-protection-shield-study.git"
   },
   "scripts": {
     "build": "bash ./bin/xpi.sh",
-    "eslint": "eslint addon test --ext=js,jsm,json",
+    "eslint": "eslint . --ext=js,jsm,json",
```

Comment:
Turns out, by linting everything, we caught a few minor nits in ./run-firefox.js (see below for the thrilling results!)